.github/workflows/windows.yml

```
@@ -17,7 +17,7 @@ concurrency:
 
 env:
   PYTHON_VERSION: '3.11'
-  TARGET_BRANCH: 'master'
+  TARGET_BRANCH: 'releases/2025/1'
```

```suggestion
  TARGET_BRANCH: ${{ github.base_ref || github.event.merge_group.base_ref || github.ref }}
```
@mryzhov should we use it similar to OpenVINO GHA code?

---

Yes, it is better to use same expression (we use it to save/restore caches in genai now or clone OV repo when needed_

---

it is better to change it in master first, not necessary to change it in this PR